src/main/java/biz/paluch/spinach/output/JobListOutput.java

```diff
@@ -49,14 +53,25 @@ public void set(ByteBuffer bytes) {
             return;
         }
 
-        body = codec.decodeValue(bytes);
+        if (body == null) {
+            counters = new HashMap<String, Long>();
```

Comment:
Initializing the `counters` map eagerly (e. g. field level) prevents a null-map. WDYT?